content/en/armory-enterprise/quickspin/install-quick-spin.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Install Quick Spin 
+linkTitle: "Install Quick Spin"
+no_list: true
+weight: 4
+description: >
+  Use the Armory Quick Spin container to quickly get an evaluation or testing instance of Spinnaker up and running.
```

Comment:
For SEO, this page can't have the same description as what's in the _index.md file.